backend/src/Squidex.Domain.Apps.Entities/Contents/DomainObject/Guards/GuardContent.cs

```diff
@@ -50,6 +48,11 @@ public static async Task CanCreate(CreateContent command, IContentWorkflow conte
         {
             Guard.NotNull(command, nameof(command));
 
+            if (!HasPermission(content, command, Permissions.AppContentsUpdate))
+            {
+                throw new DomainForbiddenException(T.Get("common.errorNoPermission"));
```

Comment:
Please do not provide a message here, it is not used anyway.

Reply:
but DomainForbiddenException must to have params string

Reply:
Thats true, but then remember to translate it as well.

Reply:
But I'll get error. I must to pass any string there. What you mean **do not provide a message here, it is not used anyway**